packages/devextreme/js/core/utils/template_manager.js

```diff
@@ -53,6 +54,10 @@ export const addOneRenderedCall = (template) => {
     const render = template.render.bind(template);
     return extend({}, template, {
         render(options) {
+            if(options && options.container) {
```

Comment:
I don't know, if patching options inside core utils functions is considered ok in DevExtreme, fix seems pretty safe, but we must doublecheck, that it doesn't have additional sideeffects
Also, are you sure, that putting this inside addOneRenderedCall function will cover all the cases, where we need to getPublicElement? For example, if isAsyncTemplate === true, addOneRenderedCall will not be executed

Reply:
Nope, I missed the **isAsyncTemplate** case. **addOneRenderedCall** is indeed not called in case of **isAsyncTemplate === true**. I used a different solution.